src/inet/IPEndPointBasis.cpp

```diff
@@ -1305,7 +1305,8 @@ INET_ERROR IPEndPointBasis::GetEndPoint(nw_endpoint_t & aEndPoint, const IPAddre
     // we want if the locale endpoint is IPv4.
     if (aAddressType == kIPAddressType_IPv4 && aAddress.Type() == kIPAddressType_Any)
     {
```

Comment:
LGTM. I'd tweak slightly and `const` qualify `anyAddr`.

Reply:
Done. At some point I will stop to forget to use `const`...